{"name": "Тестируем ошибку повторного создания курьера с теми же данными. Ручка /v1/courier", "status": "passed", "start": 1758393449727, "stop": 1758393451039, "uuid": "717707a5-f19a-4667-9c67-6e4911b042c0", "historyId": "4114455b6076625b5fd94ce51d9b214f", "testCaseId": "4114455b6076625b5fd94ce51d9b214f", "fullName": "tests.test_create_courier.TestCreateNewCourier#test_create_courier_duplicate", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_courier"}, {"name": "subSuite", "value": "TestCreateNewCourier"}, {"name": "host", "value": "NIK-T14"}, {"name": "thread", "value": "20952-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_courier"}], "titlePath": ["tests", "test_create_courier.py", "TestCreateNewCourier"]}